src/host/ToneGenerator.cpp

```diff
@@ -7,12 +7,9 @@ namespace host {
     static double angle = 0.0;
 
     for(int i = 0; i < length; i++) {
-      *stream++ = 255 * std::cos(angle);
-      angle += 3.14159 / 100;
-
-      if(angle > 2.0 * 3.14159) {
-        angle -= 2.0 * 3.14159;
-      }
+      int8_t sample = std::sin(angle) >= 0 ? 1 : -1;
+      *stream++ = 5 * sample;
+      angle += 2 * M_PI / 800;
```

Comment:
I updated, it is defined earlier in the SDL header included in the ToneGenerator.hpp